llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1840,9 +1849,12 @@ PassBuilder::buildLTODefaultPipeline(OptimizationLevel Level,
     MPM.addPass(CalledValuePropagationPass());
   }
 
-  // Now deduce any function attributes based in the current code.
-  MPM.addPass(
-      createModuleToPostOrderCGSCCPassAdaptor(PostOrderFunctionAttrsPass()));
+  // For higher optimization levels this Pass has just run, so don't repeat it.
```

Comment:
I don't think this is needed, LTO at `-O1` is not really something anybody cares about.

Reply:
Ah ok - I have updated to remove running `PostOrderFunctionAttrsPass()` here at `-O1`.